module/migration.js

```diff
@@ -286,25 +278,18 @@ export const migrateItemData = function (item) {
  * @return {Object}       The updateData to apply
  */
 export const migrateSceneData = function (scene) {
-  const tokens = foundry.utils.deepClone(scene.tokens);
```

Comment:
This caused the migration to fail on `update` because of recursive structure. Used toJSON to clone only the data.